src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -574,10 +574,16 @@ private function registerAssetsConfiguration(array $config, ContainerBuilder $co
     {
         $loader->load('assets.xml');
 
+        $removeGitStrategy = true;
```

Comment:
this logic is not needed: since the service is private, it will be removed automatically when the container is compiled